znai-enterprise/pom.xml

```diff
@@ -59,5 +59,11 @@
             <groupId>io.vertx</groupId>
             <artifactId>vertx-web</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.6</version>
```

Comment:
same as above, let's move this dep to dependency management section of the parent pom